src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/StoreKeeperSeoPages.php

```diff
@@ -0,0 +1,220 @@
+use StoreKeeper\WooCommerce\B2C\I18N;
+
+class StoreKeeperSeoPages implements WithHooksInterface
+{
+    public function registerHooks(): void
+    {
+        if (StoreKeeperSeo::isSelectedHandler()) {
+            add_filter('woocommerce_product_data_tabs', [$this, 'addSeoProductTab']);
+            add_action('woocommerce_product_data_panels', [$this, 'renderProductTab']);
+            add_action('woocommerce_admin_process_product_object', [$this, 'saveProductSeo']);
+
+            add_action('product_cat_add_form_fields', [$this, 'renderCategoryCreateFields'], 10, 2);
+            add_action('product_cat_edit_form_fields', [$this, 'renderCategoryEditFields'], 10, 1);
+            add_action('edited_product_cat', [$this, 'saveCategorySeo'], 10, 1);
+            add_action('create_product_cat', [$this, 'saveCategorySeo'], 10, 1);
+
+            add_action('admin_enqueue_scripts', function () {
+                wp_enqueue_style('storekeeper-seo-admin', plugin_dir_url(__FILE__).'../static/seo.css');
+            });
+        }
+    }
+
+    public function addSeoProductTab($tabs): array
+    {
+        $tabs['storekeeper_seo'] = [
+            'label' => __('StoreKeeper Seo', I18N::DOMAIN),
+            'target' => 'storekeeper_seo_options',
+        ];
+
+        return $tabs;
+    }
+
+    public function renderCategoryCreateFields(): void
+    {
+        $this->renderCategoryAddTextField(StoreKeeperSeo::SEO_TITLE, __('Seo title', I18N::DOMAIN));
+        $this->renderCategoryAddTextField(StoreKeeperSeo::SEO_KEYWORDS, __('Seo keywords', I18N::DOMAIN));
+        $this->renderCategoryAddTextArea(StoreKeeperSeo::SEO_DESCRIPTION, __('Seo description', I18N::DOMAIN));
+    }
+
+    public function renderCategoryEditFields($term): void
+    {
+        $attributes = [];
+        $storekeeper_id = get_term_meta($term->term_id, 'storekeeper_id', true);
+        $seo = StoreKeeperSeo::getCategorySeo($term);
+        if (!empty($storekeeper_id)) {
+            $attributes += ['disabled' => 'disabled'];
+        }
+        $this->renderCategoryEditTextField(
+                StoreKeeperSeo::SEO_TITLE,
+                __('Seo title', I18N::DOMAIN),
+                $seo[StoreKeeperSeo::SEO_TITLE],
+                $attributes
+        );
+        $this->renderCategoryEditTextField(
+            StoreKeeperSeo::SEO_KEYWORDS,
+            __('Seo keywords', I18N::DOMAIN),
+            $seo[StoreKeeperSeo::SEO_KEYWORDS],
+            $attributes
+        );
+
+        $this->renderCategoryEditTextArea(
+                StoreKeeperSeo::SEO_DESCRIPTION,
+                __('Seo description', I18N::DOMAIN),
+                $seo[StoreKeeperSeo::SEO_DESCRIPTION],
+                $attributes
+        );
+    }
+
+    public function saveCategorySeo($term_id)
+    {
+        $term = get_term($term_id);
+        $storekeeper_id = get_term_meta($term->term_id, 'storekeeper_id', true);
+        if (empty($storekeeper_id)) {
```

Comment:
Just a question, why do we only save it when no `storekeeper_id`?

Reply:
if storekeeper_id is set than it should be managed in storekeeper, not in WooCommerce